torch/csrc/inductor/aot_inductor_model_container.h

```diff
@@ -6,6 +6,30 @@
 
 #include <torch/csrc/inductor/aot_inductor_model.h>
 
+// At codegen time, we write out a binary file called constants.bin.
+// We then turn the raw binary to an object file that exposes this
+// symbol and link it into the final .so.
+// For information on the binary format, see `man objcopy`, under
+// the "binary-architecture" flag:
+// https://man7.org/linux/man-pages/man1/objcopy.1.html
+// todo: use #embed in C++ 23 once available
+extern const uint8_t _binary_constants_bin_start[];
+extern const uint8_t _binary_constants_bin_end[];
+
+#define GPU_ALIGNMENT 64
```

Comment:
This seems very likely to override some other constant elsewhere and does not need to be such a generic name or in the header